packages/e2e/nightwatch/browserstack.conf.js

```diff
@@ -0,0 +1,56 @@
+const browserstack = require('browserstack-local');
```

Comment:
Not needed?

Reply:
It seems this variable is not used here. Is it an unneeded require or does it setup some global stuff? Please try without this require. If it fails then maybe add a comment and eslint-ignore it.